docs/notebook_preamble.py

```diff
@@ -48,57 +42,10 @@ def format_dict(x):
 
 # Create an API access token
 
-claims = {
-    "daily_location": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2"],
-    },
-    "modal_location": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2"],
-    },
-    "flows": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "location_event_counts": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "meaningful_locations_aggregate": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "meaningful_locations_between_label_od_matrix": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "geography": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "unique_subscriber_counts": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "location_introversion": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "total_network_objects": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "aggregate_network_objects": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-    "radius_of_gyration": {
-        "permissions": {"run": True, "poll": True, "get_result": True},
-        "spatial_aggregation": ["admin3", "admin2", "admin1"],
-    },
-}
-
-TOKEN = make_token(
-    username="testuser", secret_key="secret", lifetime=timedelta(days=1), claims=claims
+
+TOKEN = generate_token(
```

Comment:
I trust this works, but I can't see where `generate_token()` is imported in the preamble or notebook?

Reply:
Looks like it isn’t actually- nice catch!